docs/developer/mesh.md

```diff
@@ -0,0 +1,270 @@
+# Mesh
+
+Here we describe the two mesh formarts that are used in RDycore. The two mesh
+formats that we use are:
+
+1. Exodus II
+2. DMPlex HDF5 v3.0.0
+
+RDycore has been tested for meshes that include triangular, quadrilateral, or
+both triangular and quadrilateral cells. The vertices of cells have coordinates
```

Comment:
How about: The cell vertices have three-dimensional coordinates that incorporate topographic information.